eng/Versions.props

```diff
@@ -25,10 +25,10 @@
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.132-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
     <!-- CodeStyleAnalyzerVersion should we updated together with version of dotnet-format in dotnet-tools.json -->
     <CodeStyleAnalyzerVersion>4.2.0-2.final</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>17.2.3194</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>17.3.37-preview</VisualStudioEditorPackagesVersion>
```

Comment:
Would this break integration tests?

Reply:
It did in the other PR I made, what can I do to fix that?

Reply:
We'd need to reinstate -vs-deps branch, so this won't affect people works in main. I also have a change requires new Editor package.

Reply:
is this in preview1 or preview2?  Preview1 should be rolling out to machines soonish, so if its present there we can avoid a -vs-deps branch

Reply:
hmm, looks like rel/d17.3 has 17.3.41-preview, so I guess it willbe available soon?
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VS?path=%2F.corext%2FConfigs%2FVS-Platform%2FVS-Platform.packageconfig&version=GBrel%2Fd17.3&_a=contents

Reply:
I think p1 should be rolling out to machines next week, so it might be reasonable to wait